scripts/cat/skills.py

```diff
@@ -365,7 +365,7 @@ def generate_new_catskills(status, moons, hidden_skill: HiddenSkillEnum = None):
 
         new_skill.hidden = hidden_skill
 
-        # TODO: Make this nicer
+        # TODO: IS this nicer?
```

Comment:
I was planning to once I got feedback about the change. It looks better in my opinion, but that doesn't mean it is.

Seemed more logical to add it as a code comment rather than doing a PR and then add it as a PR comment.

I also see that I accidentaly capitalized the S in "Is", which make it sound rude. Sorry for that

Reply:
I get the thought behind that, but I personally am used to comments like this making it into the codebase at some points in time and then being forgotten again.

Reply:
That is a fair point, and comment is now removed. I do not know if the code itself is an improvement, so I will let the conversation remain unsolved.

Reply:
As nothing more has been said. I will resolve this conversation